8Knot/pages/index/index_layout.py

```diff
@@ -175,80 +175,78 @@
+                            style={"fontSize": 16},
+                        ),
+                        dbc.Alert(
+                            children='Please ensure that your spelling is correct. \
+                                If your selection definitely isn\'t present, please request that \
+                                it be loaded using the help button "REPO/ORG Request" \
+                                in the bottom right corner of the screen.',
+                            id="help-alert",
+                            dismissable=True,
+                            fade=True,
+                            is_open=False,
+                            color="info",
+                        ),
+                        dbc.Alert(
+                            children="List of repos",
+                            id="repo-list-alert",
+                            dismissable=True,
+                            fade=True,
+                            is_open=False,
+                            color="light",
+                            # if number of repos is large, render as a scrolling window
+                            style={"overflow-y": "scroll", "max-height": "440px"},
+                        ),
+                    ],
+                    style={
+                        "width": "50%",
+                        "paddingRight": "10px",
+                    },
+                ),
+                dbc.Button(
+                    "Search",
+                    id="search",
+                    n_clicks=0,
+                    size="md",
                 ),
-                dbc.Alert(
-                    children='Please ensure that your spelling is correct. \
-                        If your selection definitely isn\'t present, please request that \
-                        it be loaded using the help button "REPO/ORG Request" \
-                        in the bottom right corner of the screen.',
-                    id="help-alert",
-                    dismissable=True,
-                    fade=True,
-                    is_open=False,
-                    color="info",
+                dbc.Button(
+                    "Help",
+                    id="search-help",
+                    n_clicks=0,
+                    size="md",
                 ),
-                dbc.Alert(
-                    children="List of repos",
-                    id="repo-list-alert",
-                    dismissable=True,
-                    fade=True,
-                    is_open=False,
-                    color="light",
-                    # if number of repos is large, render as a scrolling window
-                    style={"overflow-y": "scroll", "max-height": "440px"},
+                dbc.Button(
+                    "Repo List",
+                    id="repo-list-button",
+                    n_clicks=0,
+                    size="md",
+                ),
+                dbc.Switch(
```

Comment:
It isn't clear in the UI whether the switch is on or not. I'd have a piece of text that turns on/off: 
"Github Bot Filter: ACTIVE"

and change the CSS for the button, adding "class_name" as "botlist-filter"

```
.botlist-filter:checked {
    background-color: #0a7ff5;
}
```